examples/getinfo.rs

```diff
@@ -1,15 +1,20 @@
 // This example only fetches and prints the node info to the standard output similarly to
 // `lncli getinfo`.
 //
-// This program accepts three arguments: address, cert file, macaroon file
-// The address must start with `https://`!
```

Comment:
This needs to be kept, way too many people are already missing this.

Reply:
Ah yeah. Great catch 👌🏿